src/test/java/io/openshift/booster/OpenShiftIT.java

```diff
@@ -54,7 +54,7 @@
     private Session session;
 
     @AwaitRoute(path = "/api/greeting")
-    @RouteURL(GREETING_NAME)
+    @RouteURL("${app.name}")
```

Comment:
Why not keep GREETING_NAME here?

Reply:
We could obviously do that :), just a matter of taste I guess

Reply:
Well, that would mean re-interpolating the value another time (performance not a big factor here, but…) and I'd rather have the interpolating done in controlled spots so that it's easier to propagate changes (for example, if we want/need to change the name of the property).

Reply:
No problem, I'll fix it

Reply:
Actually we can't do this, compilation fails since the annotation value is not a true constant

Reply:
Ah, dammit… :(

Reply:
Yeah, that does sucks...